roles/openshift_logging/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
-openshift_logging_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default(
+  'registry.access.redhat.com/openshift3/ose-' if openshift_deployment_type == 'openshift-enterprise'
+  else 'docker.io/openshift/origin-') }}"
```

Comment:
This is probably going to be a common thing we need to do. I'm wondering if we either need to start a new `common` or `defaults` role that can define some of these values, that way we aren't intertwining ourselves more into openshift_facts or openshift_common, but can still avoid having to set these defaults everywhere.

Alternatively, maybe something like this would be better as a filter_plugin. We would still need a common location to host it (but we could potentially leverage lib_openshift, or start a similar filters_openshift role)